etc/install-minio.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+download_url="https://dl.minio.io/server/minio/release/linux-amd64/minio"
```

Comment:
let's just put this in the Makefile as a dependency of the tests. I did that for gtest in libpy and it seemed pretty nice.